doc/iterations/2025.1/add_support_for_model_publication.adoc

```diff
@@ -0,0 +1,80 @@
+= Add support for model publication
+
+== Problem
+
+Sirius Web doesn't provide an **in-application** way to import models from other projects.
+The current import/export mechanisms rely on generating an interchange format file (Json, XMI, SysML), and uploading such file in a project to import it.
+
+When exported models are used as shared reusable components, it is the responsibility of the stakeholders to:
+
+- Store the shared models in an external location
+- Ensure that every user is working on the same version of the shared model
+- Coordinate to export a new version when needed, and import it in all the dependent projects
+
+These processes are error-prone, and could be more controlled if they all happened inside the application.
+
+== Key Result
+
+Sirius Web users should be able to:
+
+- Publish a model with a given version number
+- Import a published with a given version number
+
+== Solution
+
+The contextual menu of _Model_ elements in the explorer contains a new _Publish_ tool.
+This tool opens a pop-up that asks the user for a version number and comments.
+
+**Note**: Sirius Web will not allow to publish a model with the a version number that matches an already published version of the model. 
+User will have to update the version of the model if they want to re-publish it.
```

Comment:
The initial mockup we did for this popup displays the current version number and the latest one. I agree this could be improved to let the user see the list of all the published versions, potentially with the date of publication.

In any case, with the workflow approach, I think this is not a Sirius Web question anymore: the implementation of the workflow and the UI components that let the user provide the necessary variables to handle the publication will be defined in downstream applications. This is still a question for Studio publication, but I discuss it on the corresponding comment.

Reply:
Yes I agree, I still think Sirius Web should have a page listing artifacts published but a specific downstream application will propose its end users with the integration of those artifacts using a specific user interface.